src/wp-includes/kses.php

```diff
@@ -2418,6 +2426,7 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 		$css_test_string = $css_item;
 		$found           = false;
 		$url_attr        = false;
+		$rgba_attr       = false;
```

Comment:
```suggestion
		$color_attr      = false;
```

And where the variable is used throughout.